### PR TITLE
Made test 'Verify a detached signature' readable

### PR DESCRIPTION
**Pull request by piranna**
Split `publicKeyArmored` in an array of strings so it's easier to read

### CODE REVIEW

**Comment by tanx** [comment]
Thanks for the PR. More readable tests are alway good :+1: 

Comments follow...

File: test/general/signature.js [hunk 1]
```diff
@@ -549,7 +549,8 @@ describe("Signature", function() {
   it('Verify subkey revocation signature', function(done) {
     var pubKey = openpgp.key.readArmored(pub_revoked).keys[0];
 
-    var verified = pubKey.subKeys[0].revocationSignature.verify(pubKey.primaryKey, {key: pubKey.primaryKey, bind: pubKey.subKeys[0].subKey});
+    var verified = pubKey.subKeys[0].revocationSignature.verify(pubKey.primaryKey,
+    	{key: pubKey.primaryKey, bind: pubKey.subKeys[0].subKey});
```
Comment (tanx) [inline test/general/signature.js]: This newline is not really necessary IMHO.

File: test/general/signature.js [hunk 2]
```diff
@@ -567,7 +568,8 @@ describe("Signature", function() {
   it('Verify V3 certification signature', function(done) {
     var pubKey = openpgp.key.readArmored(pub_v3).keys[0];
 
-    var verified = pubKey.users[0].selfCertifications[0].verify(pubKey.primaryKey, {key: pubKey.primaryKey, userid: pubKey.users[0].userId});
+    var verified = pubKey.users[0].selfCertifications[0].verify(pubKey.primaryKey,
+    	{key: pubKey.primaryKey, userid: pubKey.users[0].userId});
```
Comment (tanx) [inline test/general/signature.js]: Same applies here.

File: test/general/signature.js [hunk 3]
```diff
@@ -632,7 +636,28 @@ describe("Signature", function() {
...
+	    'AAHNLVdoaXRlb3V0IFVzZXIgPHNhZmV3aXRobWUudGVzdHVzZXJAZ21haWwu',
+	    'Y29tPsLAXAQQAQgAEAUCUo4O2gkQ1/uT/N+/wjwAAN2cB/9gFRmAfvEQ2qz+',
+	    'WubmT2EsSSnjPMxzG4uyykFoa+TaZCWo2Xa2tQghmU103kEkQb1OEjRjpgwJ',
+	    'YX9Kghnl8DByM686L5AXnRyHP78qRJCLXSXl0AGicboUDp5sovaa4rswQceH',
+	    'vcdWgZ/mgHTRoiQeJddy9k+H6MPFiyFaVcFwegVsmpc+dCcC8yT+qh8ZIbyG',
+	    'RJU60PmKKN7LUusP+8DbSv39zCGJCBlVVKyA4MzdF5uM+sqTdXbKzOrT5DGd',
+	    'CZaox4s+w16Sq1rHzZKFWfQPfKLDB9pyA0ufCVRA3AF6BUi7G3ZqhZiHNhMP',
+	    'NvE45V/hS1PbZcfPVoUjE2qc1Ix1',
+	    '=7Wpe',
+	    '-----END PGP PUBLIC KEY BLOCK-----'
```
Comment (tanx) [inline test/general/signature.js]: Could you indent this part using two spaces like the other?

**Comment by tanx** [comment]
I made some comments. Please rebase to the current master after fixing the the things I pointed out and `git push -f ` your `patch-1` branch to initiate a new travis build. Thanks!

**Comment by bartbutler** [comment]
Closed due to inactivity/age.

**Pull request closed without merging**